### ISSUE TITLE
[BUG] this lib is trying to access camera, is this intentional, if yes then this is a big fraud..

### ISSUE THREAD

**Issue by originalkamaal** [sarbagyastha/youtube_player_flutter#888]
**Describe the bug**

**To Reproduce**

**Expected behavior**

**Screenshots**

**Technical Details:**

**Additional context**

**Comment by Drey0507** [comment]
I agree, after adding the plugin to my app, an error popped up saying it does not have permission to access the camera.
See screenshot bellow:

![image](https://github.com/sarbagyastha/youtube_player_flutter/assets/16067194/8743d7bb-088e-47ac-a816-b19086b99f82)
